maps/persistence/outreach_areas.dm

```diff
@@ -1,3 +1,6 @@
+/area/outreach/exoplanet
+	name = "Outreach"
+
```

Comment:
This area needs the area flags AREA_FLAG_IS_BACKGROUND and AREA_FLAG_EXTERNAL or blueprints break. AREA_FLAG_EXTERNAL may not be necessary, although I think shield generators may not behave.